README.md

```diff
@@ -235,13 +234,11 @@ Thanks goes to these wonderful people
   </tr>
   <tr>
     <td align="center"><a href="http://hetpatel33.github.io"><img src="https://avatars0.githubusercontent.com/u/13877514?v=4" width="100px;" alt="Het Patel"/><br /><sub><b>Het Patel</b></sub></a><br /><a href="https://github.com/codesandbox/codesandbox-client/commits?author=hetpatel33" title="Code">💻</a></td>
-    <td align="center"><a href="https://github.com/milap1296"><img src="https://avatars2.githubusercontent.com/u/19545730?v=4" width="100px;" alt="milap1296"/><br /><sub><b>milap1296</b></sub></a><br /><a href="https://github.com/codesandbox/codesandbox-client/commits?author=milap1296" title="Code">💻</a></td>
```

Comment:
@milap1296 forgot to update .all-contributorsrc in https://github.com/codesandbox/codesandbox-client/pull/2758 so I think you should also add him @SaraVieira back.

Reply:
yeah but let me get this one in first otherwhise there will be conflicts :(